Appserver/FormSubmit/AbstractFormObject.cs

```diff
@@ -27,13 +27,13 @@ public static AbstractFormObject FromTextract(TextractDocument doc)
 
         // Top Form Information
         
```

Comment:
employerSignDate needs to be trimmed
providerSignDate needs to be trimmed

Reply:
fixed those. The 'P' is in the form itself which I have PM'd you versions without the 'P'